components/Portfolio/Trading/index.tsx

```diff
@@ -10,17 +10,31 @@ import { FactoryContext, initialFactoryState } from '@context/FactoryContext';
 const TradingPortfolio: FC = () => {
     const { allFilledOrders, factoryState: { tracers } = initialFactoryState } = useContext(FactoryContext);
     const [tab, setTab] = useState(0);
+    const [fetchedTracers, setFetchedTracers] = useState<any>([]);
+
+    const fetchTracers = setTimeout(async function getInfo() {
+        const tempTracers = [];
+        for (const key of Object.keys(tracers)) {
+            tempTracers.push(tracers[key]);
+        }
+        setFetchedTracers(tempTracers);
+    }, 1000);
+
+    useEffect(() => {
+        fetchTracers;
+    });
+
```

Comment:
Does this even call the function??

Reply:
yep, every second

Reply:
i see. Also gets called whenever the component updates. Which means you are adding a lot of timeout functions will need to update this. But approved